lib/wordpress/connector.js

```diff
@@ -58,3 +59,17 @@ export function initializeWpApollo(initialState = null) {
 
   return newApolloClient
 }
+
+/**
+ * Only update when the cache value has changed.
+ *
+ * @author WebDevStudios
+ * @param {object} pageProps Props from getStaticProps().
+ * @return {object}          WP Apollo client instance.
+ */
+export function useWpApollo(pageProps) {
+  const state = pageProps[APOLLO_STATE_PROP_NAME]
+  // Use WP Apollo instance by default.
```

Comment:
I'd suggest removing this comment line as the renaming makes it clear this is the WP Apollo instance.